spec/vcr_cassettes/manageiq/providers/hawkular/middleware_manager/middleware_datasource.yml

```diff
@@ -12,7 +12,7 @@ http_interactions:
       Accept-Encoding:
       - gzip, deflate
       User-Agent:
-      - rest-client/2.0.0.rc1 (darwin15.3.0 x86_64) ruby/2.2.4p230
+      - rest-client/2.0.0.rc1 (linux-gnu x86_64) ruby/2.3.0p0
```

Comment:
I rerecord the cassette because of this:

https://github.com/ManageIQ/manageiq/pull/9870/files#diff-ab98fdea4e102ea6b778a61777936bf5L903

Now I'm fetching the Max wait time, and some URLs changed because of that.